src/pygambit/behav.pxi

```diff
@@ -324,7 +324,36 @@ class MixedBehaviorProfile:
             If `action` is a string and no action in the game has that label.
         """
         return self._action_value(self.game._resolve_action(action, 'action_value'))
-
+	
+    def realiz_prob(self, player: typing.Union[Player, str], action: Action, node: Node) -> float:
+        """Realization probability for a node.
+        
+        Parameters
+        ----------
+        player : Player or str
+            The player to get the payoff for. If a string is passed, the
+            player is determined by finding the player with that label, if any.
+        
+        action : Action or str
+            The action to get the payoff for. If a string is passed, the
+            action is determined by finding the action with that label, if any.
+        
+        node : Node
+            The node of the game tree.
+        
+        Raises
+        ------
+        MismatchError
+            If `player` is a `Player` from a different game.
+            If `action` is an `Action` from a different game.
+            If `node` is not in the same game as the profile.
+        KeyError
+            If `player` is a string and no player in the game has that label.
+            If `action` is a string and no action in the game has that label.
+            If `node` is not in the same game as the profile.
+        """
+        return RealizationProbs(self.profile, self.player, self.game._resolve_action(action, 'action_value'),  self.game._resolve_node(node, 'node'))
```

Comment:
What is this calling?  There's no function called `RealizationProbs` anywhere.

The best way to write this function is to imitate the `infoset_prob` function.  The only difference between `realiz_prob` and `infoset_prob` is that `realiz_prob` is for one node and `infoset_prob` is the sum over all nodes in an infoset.  But otherwise the structure of the code's going to be identical - it just calls the node version of `GetRealizProb` from the C++ instead of the infoset version.

The only tiny wrinkle is that when updating `gambit.pxd` you'll need to update the wrapping - `GetRealizProb` is currently for Infoset.  Probably the best thing to do is to make `GetRealizProb` refer to Node and rename the `GetRealizProb` for Infoset to `GetInfosetProb`.   If you haven't done this before in Cython, the syntax for renaming a function in a `.pxd` is like

```
double "GetInfosetProb" GetRealizProb(c_GameInfoset)
```

This will make a call to GetInfosetProb in the Cython actually call the C++ function GetRealizProb (i.e. the bit in quotes is an alias).